traveler-ios-ui/TravelerKitUI/ViewControllers/BookableAvailabilityViewController.swift

```diff
@@ -43,7 +43,7 @@ class BookableAvailabilityViewController: UIViewController {
         case (_, let vc as BookableDetailsViewController):
             vc.errorContext = errorContext
             vc.bookingContext = bookingContext
-        case (_, let vc as BookableConfirmationViewController):
+        case (_, let vc as BookablePassesViewController):
             vc.passes = passes
             vc.product = bookingContext?.product
```

Comment:
Lets get rid of the booking context since it has no use any more. Instead of implementing booking context, lets use delegates for communication between bookablepassesVC and bookableDetailsVC.